tools/api-markdown-documenter/src/documentation-domain/PlainTextNode.ts

```diff
@@ -36,6 +36,7 @@ export class PlainTextNode extends DocumentationLiteralNodeBase<string> {
 		return this.value.length === 0;
 	}
 
+	// TODO: remove this
 	/**
```

Comment:
Was this meant to be done in this PR, or is this intended for another PR? If another PR, is there any work item tracking it?

(Non-blocking comment. I just worry about TODO comments like this because context gets lost over time and someone else might come along one day and find this and wonder.)

Reply:
I forgot about this! Haha. It's unrelated, so it should probably go in a separate PR. But I'll do that now.